diff --git a/drivers/media/dvb-frontends/si2168.c b/drivers/media/dvb-frontends/si2168.c
index 4b6e0bc..432a8b5 100644
--- a/drivers/media/dvb-frontends/si2168.c
+++ b/drivers/media/dvb-frontends/si2168.c
@@ -256,7 +256,7 @@ static int si2168_set_frontend(struct dvb_frontend *fe)
 		if (c->symbol_rate < 6000000) {
 			delivery_system = 0x10;
 			c->delivery_system = SYS_DVBC_ANNEX_B;
-			c->bandwidth_hz = 6000000;
+//			c->bandwidth_hz = 6000000;
 		}
 		break;
 	case SYS_DVBT2:
